### PR TITLE
trac#282: small test that checks possibility of placing comment before r...

### PR DESCRIPTION
**Pull request by jan-cerny**
...oot element in OVAL files

### CODE REVIEW

**Comment by isimluk** [comment]
You put `test_comment.xml` into `EXTRA_DIST`, but the filename is `comment.xml`. That means `make distchech` will probably fail.

Please make sure `make check` and `make distcheck` passes before contributing.

**Comment by jan-cerny** [comment]
Fixed, make check and make distcheck passes succesfully.

File: tests/API/OVAL/unittests/test_comment.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
```
Comment (mpreisler) [inline tests/API/OVAL/unittests/test_comment.sh]: Extra space in shebang

File: tests/API/OVAL/unittests/comment.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Comments before the root element shouldn't confuse the OVAL parser -->
+<oval_definitions xsi:schemaLocation="http://oval.mitre.org/XMLSchema/oval-definitions-5 oval-definitions-schema.xsd      http://oval.mitre.org/XMLSchema/oval-definitions-5#unix unix-definitions-schema.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5" xmlns:oval-def="http://oval.mitre.org/XMLSchema/oval-definitions-5" xmlns:oval="http://oval.mitre.org/XMLSchema/oval-common-5" xmlns:unix-def="http://oval.mitre.org/XMLSchema/oval-definitions-5#unix">
+  <generator>
+    <oval:schema_version>5.10</oval:schema_version>
+    <oval:timestamp>2009-01-12T10:41:00-05:00</oval:timestamp>
+  </generator>
+  <definitions>
+    <definition id="oval:org.mitre.oval.test:def:282" version="3" class="miscellaneous">
```
Comment (mpreisler) [inline tests/API/OVAL/unittests/comment.xml]: Not sure if it's wise to have org.mitre in testing bogus content made by us.

What do you think?

**Comment by jan-cerny** [comment]
Fixed extra whitespace in shebang, but the same problem is also in other tests:
[jcerny@t440s openscap]$ git grep "#! /bin/bash"
schemas/schematron_howto.txt:#! /bin/bash
tests/API/OVAL/unittests/test_anyxml.sh:#! /bin/bash
tests/API/OVAL/unittests/test_applicability_check.sh:#! /bin/bash
tests/API/OVAL/unittests/test_count_function.sh:#! /bin/bash
tests/API/OVAL/unittests/test_deprecated_def.sh:#! /bin/bash
tests/API/OVAL/unittests/test_empty_filename.sh:#! /bin/bash
tests/API/OVAL/unittests/test_item_not_exist.sh:#! /bin/bash

**Comment by jan-cerny** [comment]
changed IDs

**Comment by isimluk** [comment]
Thank You!

**Pull request merged**